scripts/bulk_update_leads_info.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import re
+import argparse
+import csv
+import logging
+from closeio_api import Client as CloseIO_API, APIError
+
+
+def get_contact_info(contact_no, csv_row, what, contact_type):
+    columns = [x for x in csv_row.keys()
+               if re.match(r'contact%s_%s[0-9]' % (contact_no, what), x) and csv_row[x]]
+    contact_info = []
+    for col in columns:
+        contact_info.append({what: csv_row[col], 'type': contact_type})
+    return contact_info
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('csvfile', type=argparse.FileType('rU'), help='csv file')
+parser.add_argument('--api_key', '-k', required=True, help='API Key')
+parser.add_argument('--development', '-d', action='store_true',
+                    help='Use a development (testing) server rather than production.')
+parser.add_argument('--confirmed', '-c', action='store_true',
+                    help='Without this flag, the script will do a dry run without actually updating any data.')
+parser.add_argument('--create-custom-fields', '-C', action='store_true',
+                    help='Create new custom fields, if not exists.')
+parser.add_argument('--disable-create', '-e', action='store_true',
+                    help='Prevent new lead creation. Update only exists leads.')
+args = parser.parse_args()
```

Comment:
could you also add a -h or a --fields option to print out the list/pattern of fields names that this script accepts, since the user won't know how it works otherwise?